client/src/app/site/pages/organization/pages/accounts/pages/account-detail/components/account-add-to-meetings/account-add-to-meetings.component.ts

```diff
@@ -0,0 +1,164 @@
+import { AfterViewInit, ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { UntypedFormBuilder } from '@angular/forms';
+import { Router } from '@angular/router';
+import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { Id } from 'src/app/domain/definitions/key-types';
+import { OML } from 'src/app/domain/definitions/organization-permission';
+import { AssignMeetingsResult } from 'src/app/gateways/repositories/users';
+import { MeetingControllerService } from 'src/app/site/pages/meetings/services/meeting-controller.service';
+import { ViewMeeting } from 'src/app/site/pages/meetings/view-models/view-meeting';
+import { ViewUser } from 'src/app/site/pages/meetings/view-models/view-user';
+import { OpenSlidesRouterService } from 'src/app/site/services/openslides-router.service';
+import { OperatorService } from 'src/app/site/services/operator.service';
+import { UserControllerService } from 'src/app/site/services/user-controller.service';
+import { BaseUiComponent } from 'src/app/ui/base/base-ui-component';
+
+@Component({
+    selector: `os-account-add-to-meetings`,
+    templateUrl: `./account-add-to-meetings.component.html`,
+    styleUrls: [`./account-add-to-meetings.component.scss`],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AccountAddToMeetingsComponent extends BaseUiComponent implements OnInit, AfterViewInit {
+    public user: ViewUser | null = null;
+    public canManage = false;
+    public isOwnPage = false;
+
+    public meetingsSubject = new BehaviorSubject<ViewMeeting[]>([]);
+
+    public get warningMessage(): string {
+        if (!this.selectedMeetings?.length) {
+            return _(`No meetings have been selected.`);
+        }
+        if (!this.groupName) {
+            return _(`No group name has been entered.`);
+        }
+        return null;
+    }
+
+    public get meetingsObservable(): Observable<ViewMeeting[]> {
+        return this.meetingsSubject.asObservable();
+    }
+
+    public assignMeetingsForm = this.formBuilder.group({
+        group_name: [``],
+        meeting_ids: [[]]
+    });
+
+    public get selectedMeetings(): number[] {
+        return this.assignMeetingsForm.get(`meeting_ids`).value as number[];
+    }
+
+    public get groupName(): string {
+        return this.assignMeetingsForm.get(`group_name`).value as string;
+    }
+
+    public get hasResult(): boolean {
+        let dataLength = 0;
+        Object.keys(this.resultsTableDataSubjects).forEach(key => {
+            dataLength = dataLength + this.resultsTableDataSubjects[key].value.length;
+        });
+        return !!dataLength;
```

Comment:
```suggestion
        return Object.values(this.resultsTableDataSubjects).some(subject => subject.value.length);
```
this or something similar would be shorter (not tested)